support-frontend/conf/CODE.public.conf

```diff
@@ -17,7 +17,7 @@ paymentApi.url="https://payment.code.dev-guardianapis.com"
+play.filters.headers.contentSecurityPolicy = "default-src 'self' https://ccpa-pm.sp-prod.net https://cdn.privacy-mgmt.com https://gdpr-tcfv2.sp-prod.net https://ccpa-service.sp-prod.net https://ccpa-notice.sp-prod.net https://sourcepoint.theguardian.com https://ccpa.sp-prod.net https://services.postcodeanywhere.co.uk https://stripe-intent-code.support.guardianapis.com www.paypalobjects.com t.paypal.com/ members-data-api.code.dev-theguardian.com metric-push-api-code.support.guardianapis.com www.paypal.com www.sandbox.paypal.com js.stripe.com https://payments-sandbox.amazon.com https://api-cdn.amazon.com static-na.payments-amazon.com pasteup.guim.co.uk ophan.theguardian.com j.ophan.co.uk media.guim.co.uk uploads.guim.co.uk www.google-analytics.com www.googletagmanager.com tagmanager.google.com assets.guim.co.uk www.googleadservices.com googleads.g.doubleclick.net www.google.com www.google.co.uk optimize.google.com static.ads-twitter.com bat.bing.com bid.g.doubleclick.net t.co analytics.twitter.com stats.g.doubleclick.net www.youtube-nocookie.com connect.facebook.net www.facebook.com secure.adnxs.com checkout.stripe.com fonts.googleapis.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com sentry.io www.dwin1.com *.quantummetric.com data: wss: 'unsafe-inline' q.stripe.com payment.code.dev-guardianapis.com https://interactive.guim.co.uk/ https://www.theguardian.com/ https://theguardian.com/ https://contribution-referrals-code.support.guardianapis.com/referral-code; frame-ancestors https://gnmtouchpoint--dev1--c.cs88.visual.force.com https://*.lightning.force.com; worker-src blob:; child-src https://www.facebook.com/ https://www.google.com/ https://js.stripe.com/ https://cdn.privacy-mgmt.com/ blob:;"
```

Comment:
It's so impossible to read the diff on the CSP, could you flag the changes please (just in a github comment is fine)

Reply:
I agree regarding CSP changes being difficult to diff, I'm going to create a card to see if it's possible to split the string across multiple lines in the source code, this would make it far easier to manage/review! 

Regarding the changes to the CSP in this PR, we've included a new source `*.quantummetric.com` under the `default-src` directive. We also had to explicitly include 3 new directives `worker-src`, `child-src` and `frame-src`, because we're explicitly including these now we've had to duplicate some of the sources previously covered by `default-src` directive. 

This diff illustrates the difference...

<img width="1365" alt="Screenshot 2021-11-09 at 12 26 38" src="https://user-images.githubusercontent.com/1590704/140924387-24c7d600-a334-4360-8494-63863cea2475.png">

Reply:
The only change to the CSPs is the addition of `*.quantummetric.com` and `blob:` to the `default-src` directive.